src/storage/rocksdb/src/main/java/org/locationtech/geogig/rocksdb/RocksdbObjectStore.java

```diff
@@ -419,7 +419,7 @@ public final void putAll(Iterator<? extends RevObject> objects, final BulkOpList
     protected void putAll(Stream<RevObject> stream, BulkOpListener listener) {
         final Stopwatch sw = LOG.isTraceEnabled() ? Stopwatch.createStarted() : null;
 
-        final int batchsize = 10_000;
+        final int batchsize = 1000;
```

Comment:
did we do any performance testing with this.  I think there's some matching # in the GeoTools/GeoServer feature iterators....

Reply:
I did empitically, although I've work in progress in the `geogig-benchmarks` project to do proper rocks backend performance testing